codec-redis/src/main/java/io/netty/handler/codec/redis/AggregatedHeaderRedisMessage.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.util.internal.StringUtil;
+import io.netty.util.internal.UnstableApi;
+
+/**
+ * Header of Redis Aggregated types Message.
+ */
+@UnstableApi
+public abstract class AggregatedHeaderRedisMessage implements RedisMessage {
+
+    private final long length;
+
+    /**
+     * Creates a {@link AggregatedHeaderRedisMessage} for the given {@code length}.
+     */
+    public AggregatedHeaderRedisMessage(long length) {
```

Comment:
abstract classes should have a protected constructor

Reply:
Yes, i had fixed it